tests-legacy/resources/ModulesOverrideInstallUninstallTest/AdminProductsController.php

```diff
@@ -1,280 +1,11 @@
-        }
-        $this->_group = 'GROUP BY '.$alias.'.id_product';
-        $this->fields_list = array();
-        $this->fields_list['id_product'] = array(
-            'title' => $this->l('ID'),
-            'align' => 'center',
-            'class' => 'fixed-width-xs',
-            'type' => 'int',
-        );
-        $this->fields_list['image'] = array(
-            'title' => $this->l('Image'),
-            'align' => 'center',
-            'image' => 'p',
-            'orderby' => false,
-            'filter' => false,
-            'search' => false,
-        );
-        $this->fields_list['name'] = array(
-            'title' => $this->l('Name'),
-            'filter_key' => 'b!name',
-        );
-        $this->fields_list['reference'] = array(
-            'title' => $this->l('Reference'),
-            'align' => 'left',
-        );
-        if (Shop::isFeatureActive() && Shop::getContext() != Shop::CONTEXT_SHOP) {
-            $this->fields_list['shopname'] = array(
-                'title' => $this->l('Default shop'),
-                'filter_key' => 'shop!name',
-            );
-        } else {
-            $this->fields_list['name_category'] = array(
-                'title' => $this->l('Category'),
-                'filter_key' => 'cl!name',
-            );
-        }
-        $this->fields_list['price'] = array(
-            'title' => $this->l('Base price'),
-            'type' => 'price',
-            'align' => 'text-right',
-            'filter_key' => 'a!price',
-        );
-        $this->fields_list['price_final'] = array(
-            'title' => $this->l('Final price'),
-            'type' => 'price',
-            'align' => 'text-right',
-            'havingFilter' => true,
-            'orderby' => false,
-            'search' => false,
-        );
-        if (Configuration::get('PS_STOCK_MANAGEMENT')) {
-            $this->fields_list['sav_quantity'] = array(
-                'title' => $this->l('Quantity'),
-                'type' => 'int',
-                'align' => 'text-right',
-                'filter_key' => 'sav!quantity',
-                'orderby' => true,
-                'badge_danger' => true,
-            );
-        }
-        $this->fields_list['active'] = array(
-            'title' => $this->l('Status'),
-            'active' => 'status',
-            'filter_key' => $alias.'!active',
-            'align' => 'text-center',
-            'type' => 'bool',
-            'class' => 'fixed-width-sm',
-            'orderby' => false,
-        );
-        if ($join_category && (int)$this->id_current_category) {
-            $this->fields_list['position'] = array(
-                'title' => $this->l('Position'),
-                'filter_key' => 'cp!position',
-                'align' => 'center',
-                'position' => 'position',
-            );
-        }
-    }
-
     /*
     * module: pscsx32412
-    * date: 2015-07-13 16:05:31
+    * date: 2018-12-26 14:14:05
```

Comment:
What's this?

Reply:
This is something which is added when you add override in your module.